### DISCUSSION TITLE
Theme Requests

### DISCUSSION THREAD

**Discussion by saberzero1** [saberzero1/quartz-themes#6] [Ideas]
## Theme Requests

Please leave Obsidian Theme requests here, preferably with a link to the theme.

**Comment by saberzero1** [comment]
[ITS theme](https://github.com/SlRvb/Obsidian--ITS-Theme)

**Reply by simonhebich** [reply]
especially with the [alternative dnd color scheme](https://publish.obsidian.md/slrvb-docs/ITS+Theme/Alt+Color+Schemes/WOTC-Beyond)
i tried implementing it myself but i failed miserably haha

**Comment by saberzero1** [comment]
Closing this discussion as all Obsidian themes are being implemented.
